src/sm/map_icons.asm

```diff
@@ -6,12 +6,11 @@ Return_From_Hijack:
 org $C2FB80 ; Beginning of a large block of free space
 base $82FB80
 Map_Labels_Hijack:
-PHA ; getting the keysanity flag in A, so let's preserve it first
-LDA config_keysanity ; check to see if this is keysanity
+LDA $80FF52 ; load config bits
```

Comment:
Similar thing here as in keycards.asm, not a big fan of using the seed identifier fields for this.